frontend/src/metabase/setup/components/SettingsPage/SettingsPage.tsx

```diff
@@ -12,11 +13,13 @@ import { UsageQuestionStep } from "../UsageQuestionStep";
 import { PageBody, PageHeader } from "./SettingsPage.styled";
 
 export const SettingsPage = (): JSX.Element => {
+  const usageReason = useSelector(s => s.setup.usageReason);
+
   const numberedSteps = [
```

Comment:
Those steps will be moved to a custom selector, that will be used also the the thunks or reducers to check which step is the "next" to go after a submit